sklearn/metrics/tests/test_ranking.py

```diff
@@ -114,16 +114,13 @@ def _average_precision(y_true, y_score):
     y_true = y_true[order]
 
     score = 0
+    cumul_prec = 0
     for i in range(len(y_score)):
         if y_true[i] == pos_label:
             # Compute precision up to document i
             # i.e, percentage of relevant documents up to document i.
-            prec = 0
```

Comment:
What is this change about?

Reply:
just readability + efficiency (but the gain will not be noticeable, it is a fast loop). Rewriting it like that helped me to understand this code.

Reply:
The point is to have an implementation in the test that looks almost stupidly like the formulas in order to ensure correctness of a more sophisticated implementation. I think it is safer if you leave this code alone in the present PR

Reply:
ok